src/parse/mark.js

```diff
@@ -23,6 +36,17 @@ function parseMark(model, mark) {
   return mark;
 }
 
+// set color given in graph if "default" specified in spec for marks color
+function defaultColor(property, configProperty, prop, config) {
+  if (property in prop && 'value' in prop[property] && prop[property]['value'] === 'default') {
+    if (typeof config !== 'undefined' && 'marks' in config && configProperty in config.marks) {
+      prop[property]['value'] = config.marks[configProperty];
+    } else {
+      prop[property]['value'] = '#000000';
+    }
+  }
+}
+
```

Comment:
@emilygu: A couple of questions here:

1. Is there a reason you named this property `config.marks.defaultFill` rather than simply `config.marks.fill`? 
2. Is there a reason they are only applied when the property is set to `{"value": "default"}` rather than automatically applied if no fill color is specified? I would think that if no fill color is specified, the default one is automatically used; and if users explicitly want _no_ fill color specified, and have a theme applied, they can set the fill to `transparent`?